### PR TITLE
Change check version script to remove the ODFE dependency in finding the plugins

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Change check version to remove the ODFE dependency in finding the plugins

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/589754884
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/589754891
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/589754892

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: .github/workflows/staging-build-tar.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
 #    - cron: '0 10 * * *'
   repository_dispatch:
     types: [staging-build-tar]
-
+
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-tar.yml:8]: Remove the additional space right here.

File: .github/workflows/staging-test-tar-x64.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ on:
   repository_dispatch:
     types: [staging-test-tar-x64]
 
+
```
Comment (peterzhuamazon) [inline .github/workflows/staging-test-tar-x64.yml:9]: Remove the additional newline right here.

File: elasticsearch/docker/build/elasticsearch/bin/docker-entrypoint.sh [hunk 3]
```diff
@@ -88,15 +88,15 @@ if [[ -d "/usr/share/elasticsearch/plugins/opendistro_security" && "$DISABLE_INS
...
     CLK_TCK=`/usr/bin/getconf CLK_TCK`
     ES_JAVA_OPTS="-Dclk.tck=$CLK_TCK -Djdk.attach.allowAttachSelf=true $ES_JAVA_OPTS"
     if [[ -d "/usr/share/elasticsearch/performance-analyzer-rca" ]]; then
         ES_JAVA_OPTS="-Djava.security.policy=file:///usr/share/elasticsearch/performance-analyzer-rca/pa_config/es_security.policy $ES_JAVA_OPTS"
         /usr/bin/supervisord -c /usr/share/elasticsearch/performance-analyzer-rca/pa_config/supervisord.conf
     else
-        ES_JAVA_OPTS="-Djava.security.policy=file:///usr/share/elasticsearch/plugins/opendistro_performance_analyzer/pa_config/es_security.policy $ES_JAVA_OPTS"
-        /usr/bin/supervisord -c /usr/share/elasticsearch/plugins/opendistro_performance_analyzer/pa_config/supervisord.conf
+        ES_JAVA_OPTS="-Djava.security.policy=file:///usr/share/elasticsearch/plugins/opendistro-performance-analyzer/pa_config/es_security.policy $ES_JAVA_OPTS"
+        /usr/bin/supervisord -c /usr/share/elasticsearch/plugins/opendistro-performance-analyzer/pa_config/supervisord.conf
```
Comment (peterzhuamazon) [inline elasticsearch/docker/build/elasticsearch/bin/docker-entrypoint.sh:99]: Can you change this to dynamic retrieval of performance folder name instead of hard coding? Like this one I add in the Dockerfile: https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/elasticsearch/docker/templates/Dockerfile.j2#L126

File: elasticsearch/linux_distributions/opendistro-tar-build.sh [hunk 4]
```diff
@@ -107,7 +107,7 @@ chmod -R 755 $WORK_DIR/$rca_dir
 # Move agent script directly into ES_HOME/bin
 perf_analyzer=`ls -p $WORK_DIR/bin/ | grep performance`
 mv $WORK_DIR/bin/$perf_analyzer/performance-analyzer-agent-cli $WORK_DIR/bin
-rm -rf $WORK_DIR/bin/opendistro_performance_analyzer
+rm -rf $WORK_DIR/bin/opendistro-performance-analyzer
```
Comment (peterzhuamazon) [inline elasticsearch/linux_distributions/opendistro-tar-build.sh:111]: Same as above.

File: elasticsearch/linux_distributions/opendistro-tar-install.sh [hunk 5]
```diff
@@ -19,16 +19,16 @@ ES_KNN_LIB_DIR=$ES_HOME/plugins/opendistro-knn/knn-lib
...
+PA_AGENT_JAVA_OPTS="-Dlog4j.configurationFile=$ES_HOME/plugins/opendistro-performance-analyzer/pa_config/log4j2.xml \
               -Xms64M -Xmx64M -XX:+UseSerialGC -XX:CICompilerCount=1 -XX:-TieredCompilation -XX:InitialCodeCacheSize=4096 \
               -XX:InitialBootClassLoaderMetaspaceSize=30720 -XX:MaxRAM=400m"
 
 ES_MAIN_CLASS="com.amazon.opendistro.elasticsearch.performanceanalyzer.PerformanceAnalyzerApp" \
-ES_ADDITIONAL_CLASSPATH_DIRECTORIES=plugins/opendistro_performance_analyzer \
+ES_ADDITIONAL_CLASSPATH_DIRECTORIES=plugins/opendistro-performance-analyzer \
 ES_JAVA_OPTS=$PA_AGENT_JAVA_OPTS
 
 if ! grep -q '## OpenDistro Performance Analyzer' $ES_HOME/config/jvm.options; then
```
Comment (peterzhuamazon) [inline elasticsearch/linux_distributions/opendistro-tar-install.sh:34]: Same as above.

File: elasticsearch/linux_distributions/opendistro-tar-install.sh [hunk 6]
```diff
@@ -37,7 +37,7 @@ if ! grep -q '## OpenDistro Performance Analyzer' $ES_HOME/config/jvm.options; t
    echo '## OpenDistro Performance Analyzer' >> $ES_HOME/config/jvm.options
    echo "-Dclk.tck=$CLK_TCK" >> $ES_HOME/config/jvm.options
    echo "-Djdk.attach.allowAttachSelf=true" >> $ES_HOME/config/jvm.options
-   echo "-Djava.security.policy=$ES_HOME/plugins/opendistro_performance_analyzer/pa_config/es_security.policy" >> $ES_HOME/config/jvm.options
+   echo "-Djava.security.policy=$ES_HOME/plugins/opendistro-performance-analyzer/pa_config/es_security.policy" >> $ES_HOME/config/jvm.options
```
Comment (peterzhuamazon) [inline elasticsearch/linux_distributions/opendistro-tar-install.sh:40]: Same as above.

File: release-tools/scripts/check_plugin.sh [hunk 7]
```diff
@@ -142,7 +142,7 @@ do
       plugin_arch=${plugin_arch_array[$lindex]}; if [ "$plugin_arch" = "noarch" ]; then plugin_arch="";  fi
       plugin_type=${plugin_type_array[$lindex]}
       plugin_latest=`aws s3api list-objects --bucket $plugin_bucket --prefix $plugin_path --query 'Contents[].[Key]' --output text \
-                     | grep "${plugin_basename}" | grep "${ODFE_VERSION}" | grep "${plugin_platform}" | grep "${plugin_arch}" | grep "${plugin_build}" | grep "${plugin_type}" \
```
Comment (peterzhuamazon) [inline release-tools/scripts/check_plugin.sh:145]: Could you change this to grep plugin_version instead of removing it?

File: release-tools/scripts/manifest.yml [hunk 8]
```diff
@@ -5,9 +5,9 @@ versions:
...
   ODFE:
-    previous: 1.12.0
-    current: 1.13.0
-    next: 1.13.1
+    previous: 1.13.0
+    current: 1.13.1
+    next: 1.13.2
 
 urls:
   ES:
```
Comment (peterzhuamazon) [inline release-tools/scripts/manifest.yml:13]: These kind of change should be in a separate PR.

**Comment by sreekarjami** [comment]
Will open new PR.

**Comment by peterzhuamazon** [comment]
PR creator forget to pull before creating PR, close now.
